From: Dave Plater <davejplater@gmail.com>
Date: 2018-06-18 12:34:22 +0200
Subject: Something has changed in Factory
References: 
Upstream: to be submitted

Something has changed in Factory to cause the "-" to be outlawed in "name" in:
GST_PLUGIN_DEFINE(major,minor,name,description,init,version,license,package,origin)

Index: src/gst/dec/bt-dec.c
===================================================================
--- src/gst/dec/bt-dec.c.orig	2015-12-29 23:24:20.000000000 +0200
+++ src/gst/dec/bt-dec.c	2018-06-18 10:59:18.702989750 +0200
@@ -947,12 +947,12 @@ plugin_init (GstPlugin * plugin)
       bt_dec_type_find, exts, GST_CAPS_ANY, NULL, NULL);
   g_free (exts);
 
-  return gst_element_register (plugin, "buzztrax-dec", GST_RANK_MARGINAL,
+  return gst_element_register (plugin, "buzztrax_dec", GST_RANK_MARGINAL,
       BT_TYPE_DEC);
 }
 
 GST_PLUGIN_DEFINE (GST_VERSION_MAJOR,
     GST_VERSION_MINOR,
-    buzztrax - dec,
+    buzztrax_dec,
     "Buzztrax song renderer",
     plugin_init, VERSION, "LGPL", PACKAGE_NAME, "http://www.buzztrax.org");
